jsonnet/kube-prometheus/prometheus/prometheus.libsonnet

```diff
@@ -420,6 +420,11 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
                   regex: 'apiserver_admission_step_admission_latencies_seconds_.*',
                   action: 'drop',
                 },
+                {
+                  sourceLabels: ['__name__', 'le'],
+                  regex: 'apiserver_request_duration_seconds_bucket;(0.15|0.25|0.3|0.35|0.4|0.45|0.6|0.7|0.8|0.9|1.25|1.5|1.75|2.5|3|3.5|4.5|6|7|8|9|15|25|30|50)',
```

Comment:
Cannot we drop whole histogram and leave only `_count` and `_sum`? This way even when buckets change, we still won't include them.

Reply:
count and sum are in regards to the entire histogram bucket group, so it wouldn't allow us to do p99 analysis for example